### PR TITLE
Support for Python3 and Kodi 19

### PR DESCRIPTION
**Pull request by sagiben**
tested only on Kodi 19.

### CODE REVIEW

File: addon.xml [hunk 1]
```diff
@@ -1,7 +1,8 @@
-<?xml version="1.0" encoding="UTF-8" standalone="yes"?><addon id="service.subtitles.screwzira" name="ScrewZira" version="1.1.1" provider-name="CaTz">
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="service.subtitles.screwzira" name="ScrewZira" version="1.1.1" provider-name="CaTz">
   <requires>
-    <import addon="xbmc.python" version="2.14.0"/>
-    <import addon="script.module.requests" version="2.4.3"/>
+    <import addon="xbmc.python" version="2.25.0"/>
```
Comment (CaTzil) [inline addon.xml:4]: Should we specify here v 3?

**Review by CaTzil — CHANGES_REQUESTED** [review]

**Comment by CaTzil** [comment]
Thanx for the PR!

Is this change supported is by Kodi 18?

**Comment by sagiben** [comment]
Did not have time to check on Kodi 18, will try to do it later. 
But Kodi 18 should support both python2.7 and python3

**Comment by CaTzil** [comment]
I will check it later today, thank you! :]

**Comment by CaTzil** [comment]
Hi, I've tested it on kodi 18, it is fails on 
```
  File "/Users/***/Library/Application Support/Kodi/addons/service.subtitles.screwzira/service.py", line 5, in <module>
        import urllib.parse
        ImportError: No module named parse
```
Can you help with it?

**Comment by sagiben** [comment]
> Hi, I've tested it on kodi 18, it is fails on
> 
> ```
>   File "/Users/***/Library/Application Support/Kodi/addons/service.subtitles.screwzira/service.py", line 5, in <module>
>         import urllib.parse
>         ImportError: No module named parse
> ```
> 
> Can you help with it?

Sure. on it

**Pull request merged**